packages/shared-base/src/NextID/index.ts

```diff
@@ -0,0 +1,22 @@
+import { NextIDPlatform, ProfileIdentifier } from '../index.js'
```

Comment:
We should not name this file as `index.ts`.

Reply:
Maybe reason is this:
![Uploading image.png…]()